packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -565,6 +565,61 @@ describe('TransferDomain', () => {
       await expect(promise).rejects.toThrow('Non-DAT or LP tokens are not supported for transferdomain')
     })
 
+    it('(evm -> dvm) should fail if LP token is transferred', async () => {
+      const metadata = {
+        tokenA: 'DFI',
+        tokenB: 'BTC',
+        commission: 1,
+        status: true,
+        ownerAddress: dvmAddr
+      }
+      await client.poolpair.createPoolPair(metadata)
+
+      await container.generate(1)
+      const promise = client.account.transferDomain([
+        {
+          src: {
+            address: evmAddr,
+            amount: '10@DFI-BTC',
+            domain: TransferDomainType.EVM
+          },
+          dst: {
+            address: dvmAddr,
+            amount: '10@DFI-BTC',
+            domain: TransferDomainType.DVM
+          }
+        }
+      ])
+      await expect(promise).rejects.toThrow(RpcApiError)
+      await expect(promise).rejects.toThrow(
+        "RpcApiError: 'Test TransferDomainTx execution failed:\n" +
+        "Non-DAT or LP tokens are not supported for transferdomain', code: -32600, method: transferdomain"
+      )
+    })
+
+    it('(dvm -> evm) should fail if LP token is transferred', async () => {
+      // Using created pool pair from "(evm -> dvm) should fail if LP token is transferred" test
```

Comment:
As discussed, let's move the creation of poolpair in beforeAll to remove dep on previous `it`

Reply:
sounds good